functions/controllers/onboardingController.js

```diff
@@ -277,10 +277,18 @@ const advanceOnboardingStatus = functions.https.onCall(
 
 const setupUserProfile = functions.https.onCall(async (data, context) => {
   try {
-    const { uid, fullName, occupation, socialLinks, bio } = data;
+    const { uid, fullName, occupation, socialLinks, bio, profilePhotoUrl } =
+      data;
 
     // Validate fields
-    if (!uid || !fullName || !occupation || !socialLinks || !bio) {
+    if (
+      !uid ||
+      !fullName ||
+      !occupation ||
+      !socialLinks ||
+      !bio ||
+      !profilePhotoUrl
```

Comment:
Sorry for not mentioning it. profilePhotoURL doesn't need to be required, you can remove this validation